pages/api/gamelist/index.js

```diff
@@ -0,0 +1,46 @@
+import dbConnect from "../../../lib/dbConnect";
+import GameCard from "../../../models/GameCards";
+
+export default async function handler(request, response) {
+  if (request.method === "GET") {
+    await dbConnect();
+
+    const gameCard = await GameCard.find();
+    const sanitizedGameCard = gameCard.map((game) => ({
+      id: game._id.toString(),
```

Comment:
one more thing, I just saw in the automatic email I recieved when you answered: I don't think you need to convert your id now to string, it already should be a string

Reply:
Hey Mark, thanks again! deleted the toString() method. However still need the _id so that my app dont throw the key prop error at me